modules/polaris/src/main/ts/ephox/polaris/words/Words.ts

```diff
@@ -74,9 +107,15 @@ const findWordsWithIndices = <T>(chars: Word<T>, sChars: string[], characterMap:
           start: startOfWord,
           end: endOfWord
         });
+        couldBeAnAcronym.prev = undefined;
+        couldBeAnAcronym.check = undefined;
+        couldBeAnAcronym.isTheLast = false;
```

Comment:
This doesn't seem necessary, you're in an `if` block and the only other code before repeating the reset is `word = []` 🤔

Reply:
you are right, removed it